Update setup-guides-for-microsoft-365.md (#7557)
MicrosoftDocs/microsoft-365-docs: microsoft-365/enterprise/setup-guides-for-microsoft-365.md

   ![Training & guides card in the Microsoft 365 admin center](../media/setup-guides-for-microsoft-365/adminportal-trainingandguides.png)

> 3. Select **Advanced deployment guides** and clikc on **All guides**

Reviewer: ```suggestion
3. Select **Advanced deployment guides** and then select **All guides**.
```

The [Microsoft Search setup guide](https://aka.ms/MicrosoftSearchSetup) helps you configure Microsoft Search whether you want to pilot it to a group of users or roll it out to everyone in your org. You'll assign Search admins and Search editors and then customize the search experience for your users with answers and more options, like adding the Bing extension to Chrome or setting Bing as your default search engine.

## Guides for authentication and access

### Configure multifactor authentication (MFA)

> The [Configure multifactor authentication (MFA) guide](https://admin.microsoft.com/Adminportal/Home?source=applauncher#/featureexplorer/security/ConditionalAccess) provides information to Secure your organization against breaches due to lost or stolen credentials. MFA immediately increases account security by prompting for multiple forms of verification to prove a user's identity when they sign in to an app or other company resource. This prompt could be to enter a code on the user's mobile device or to provide a fingerprint scan. MFA is enabled through Conditional Access, security defaults, or per-user MFA. This guide will provide the recommended MFA option for your org, based on your licenses and existing configuration.

Reviewer: ```suggestion
The [Configure multifactor authentication (MFA) guide](https://admin.microsoft.com/Adminportal/Home?source=applauncher#/featureexplorer/security/ConditionalAccess) provides information to secure your organization against breaches due to lost or stolen credentials. MFA immediately increases account security by prompting for multiple forms of verification to prove a user's identity when they sign in to an app or other company resource. This prompt could be to enter a code on the user's mobile device or to provide a fingerprint scan. MFA is enabled through Conditional Access, security defaults, or per-user MFA. This guide will provide the recommended MFA option for your org, based on your licenses and existing configuration.
```

### Identity security for Teams

The [Identity security for teams guide](https://admin.microsoft.com/Adminportal/Home?source=applauncher#/teamsidentity) helps you with some basic security steps you can take to ensure your users are safe and have the most productive time using **Teams**.

### Add or sync users to Microsoft 365

> [This guide](https://admin.microsoft.com/Adminportal/Home?source=applauncher#/modernonboarding/identitywizard) will help streamline the process of getting your user accounts setup in **Microsoft 365**. Based on your environment and needs, you can choose to add users individually, migrate your on-premises directory with Azure AD cloud sync or Azure AD Connect, or troubleshoot existing sync problems when necessary.

Reviewer: ```suggestion
[This guide](https://admin.microsoft.com/Adminportal/Home?source=applauncher#/modernonboarding/identitywizard) will help streamline the process of getting your user accounts set up in **Microsoft 365**. Based on your environment and needs, you can choose to add users individually, migrate your on-premises directory with Azure AD cloud sync or Azure AD Connect, or troubleshoot existing sync problems when necessary.
```

Use the [Plan your passwordless deployment](https://aka.ms/passwordlesssetup) guide to discover the best passwordless authentication methods to use and receive guidance on how to deploy them.

### Integrate a third-party cloud app with Azure AD

> [This guide](https://admin.microsoft.com/Adminportal/Home?source=applauncher#/azureadappintegration) helps helps IT admins to select and configure the App.

Reviewer: ```suggestion
[This guide](https://admin.microsoft.com/Adminportal/Home?source=applauncher#/azureadappintegration) helps IT admins to select and configure the App.
```

> [This guide](https://admin.microsoft.com/Adminportal/Home?source=applauncher#/azureadappintegration) helps helps IT admins to select and configure the App.
> 

Reviewer: ```suggestion
[This guide](https://admin.microsoft.com/Adminportal/Home?source=applauncher#/azureadappintegration) helps IT admins to select and configure the App.
```

## Guides for security and compliance

### Security Analyzer

> The Microsoft security suite offers an integrated ecosystem to help protect your organization. This guide helps you analyze your security posture. The Microsoft secure score is a measurement of an org's security posture, with a higher number indicating a stronger security posture. The Analyzer 
> 

Reviewer: ```suggestion
The Microsoft security suite offers an integrated ecosystem to help protect your organization. This guide helps you analyze your security posture. The Microsoft secure score is a measurement of an organization's security posture, with a higher number indicating a stronger security posture.
```

Transform how your employees work together with the [Employee experience dashboard](https://aka.ms/EmployeeExperienceDashboard). For seamless teamwork, use Microsoft 365 to create productive, aligned teams, and keep employees engaged with leadership and the rest of the organization. Help your employees be effective in all work activities. These guides will provide instructions on how to use SharePoint, Teams, and Yammer to build collaboration across your org to help drive productivity.

### Microsoft 365 Apps setup guide

> The [Microsoft 365 Apps setup guide](https://aka.ms/OPPquickstartguide) helps you get your users' devices running the latest version of Office products like Word, Excel, PowerPoint, and OneNote. You'll get guidance on the various deployment methods that include easy self-install options to enterprise deployments with management tools. The instructions will help you assess your environment, figure out your specific deployment requirements, and implement the necessary support tools to ensure a successful install.

Reviewer: ```suggestion
The [Microsoft 365 Apps setup guide](https://aka.ms/OPPquickstartguide) helps you get your users' devices running the latest version of Office products like Word, Excel, PowerPoint, and OneNote. You'll get guidance on the various deployment methods that include easy self-install options to enterprise deployments with management tools. The instructions will help you assess your environment, figure out your specific deployment requirements, and implement the necessary support tools to ensure a successful installation.
```
